sys/include/fs/littlefs_fs.h

```diff
@@ -82,18 +85,18 @@ typedef struct {
     uint16_t sectors_per_block; /**< number of sectors per block */
```

Comment:
Please move this to the end of the struct to avoid the padding

Reply:
Will do. But note that this will not change the padding. It would just move the two padding bytes from the middle of the `struct` to its end.